pandas/plotting/_matplotlib/converter.py

```diff
@@ -246,19 +245,6 @@ def get_datevalue(date, freq):
     raise ValueError(f"Unrecognizable date '{date}'")
 
 
-def _dt_to_float_ordinal(dt):
```

Comment:
We just always use matplotlib. Will need to check / bump the minimum supported version.